content/doc/pipeline/steps/index.html.haml

```diff
@@ -16,19 +16,55 @@ title: "Pipeline Steps Reference"
   %a{:href => "/doc/book/pipeline/syntax"} Pipeline Syntax
   page.
 
+.input-group.col-lg-3.mb-3
```

Comment:
this should be wider, it looks very out of place right now

Reply:
was the previous width fine?

Reply:
I think so, maybe a little smaller would need to see it to confirm

Reply:
do I need to create a separate PR to fix this?

Reply:
> do I need to create a separate PR to fix this?

Yes, it will need a separate PR since I was too enthusiastic and merged before the review was complete.

Reply:
PR at #5279.
Sorry for the inconvenience caused.